routes/web.php

```diff
@@ -251,6 +253,24 @@
             Route::get('/archive/{id}', 'AdminDash@archiveIncident');
             Route::get('/delete/{id}', 'AdminDash@deleteIncident');
         });
+        Route::prefix('realops')->middleware('toggle:realops')->middleware('permission:staff')->group(function () {
+            Route::get('/', 'RealopsController@adminIndex');
+            Route::get('/create', 'RealopsController@showCreateFlight');
+            Route::post('/create', 'RealopsController@createFlight');
+            Route::post('/create/bulk', 'RealopsController@bulkUploadFlights');
+            Route::get('/edit/{id}', 'RealopsController@showEditFlight');
+            Route::put('/edit/{id}', 'RealopsController@editFlight');
+            Route::get('/{id}', 'RealopsController@deleteFlight');
+            Route::put('/assign-pilot/{id}', 'RealopsController@assignPilotToFlight');
+            Route::get('/remove-pilot/{id}', 'RealopsController@removePilotFromFlight');
+            Route::get('/delete/{id}', 'RealopsController@deleteFlight');
+        });
+        Route::prefix('toggles')->middleware('permission:snrStaff')->group(function () {
+            Route::get('/', 'AdminDash@showFeatureToggles');
+            Route::get('/create', 'AdminDash@showCreateFeatureToggle');
```

Comment:
Should we limit the create feature toggle function to the WM? Other staff members may not understand what this does or how to use it

Reply:
We could...I don't think it's a huge deal if someone adds a toggle on accident because it shouldn't break anything and it shouldn't be difficult to remove. But adding/removing toggles (once removing is implemented) could be limited to just the dev team.

I also meant to make the permissions open to staff and not just senior staff (so for example the EC can toggle on/off Realops). I'm wondering if git switched that with the feedback section in some of the rebasing I did. Opinions on that?

Reply:
Ok, no problem. No issue giving toggle permissions to all staff

Reply:
Resolved and squashed into 8b6c2a1